lib/vocabularies/applicator/patternProperties.ts

```diff
@@ -51,18 +59,22 @@ const def: CodeKeywordDefinition = {
     function validateProperties(pat: string): void {
       gen.forIn("key", data, (key) => {
         gen.if(_`${usePattern(cxt, pat)}.test(${key})`, () => {
-          cxt.subschema(
-            {
-              keyword: "patternProperties",
-              schemaProp: pat,
-              dataProp: key,
-              dataPropType: Type.Str,
-            },
-            valid
-          )
+          const alwaysValid = !validPatterns.includes(pat)
+          if (!alwaysValid) {
```

Comment:
... and here it's double negated again...

It would probably be better to have the list of patters that have alwaysValidSchemas and simply check that the pattern is not included there.